### PR TITLE
[IDL]: Remove Mini-cards on Data Vis Design page

### PR DESCRIPTION
**Pull request by DanaPaslaru**
[IDL]: Remove current Mini-cards on Design Basics page and replace with “Learn more on Carbon > ” as a text link in the paragraph of text.

Related to issue #651

### CODE REVIEW

File: src/pages/data-visualization/design/basics.mdx [hunk 1]
```diff
@@ -44,18 +44,8 @@ To make data visualization distinctive as IBM products, there are multiple eleme
 <Row>
 <Column colMd={5} colLg={8}>
 
-The title should reflect the main insight the data reveals. The legend should explain the chart’s meaning by defining the association of each visual property, such as color, shape and size, to the corresponding data. When possible, use labels directly on the chart to avoid long legends. All the text should be concise and easy to understand.
+The title should reflect the main insight the data reveals. The legend should explain the chart’s meaning by defining the association of each visual property, such as color, shape and size, to the corresponding data. When possible, use labels directly on the chart to avoid long legends. All the text should be concise and easy to understand. [Learn more on Carbon.](https://www.carbondesignsystem.com/data-visualization/legends)
```
Comment (vpicone) [inline src/pages/data-visualization/design/basics.mdx:47]: Should this be "Learn more **at** Carbon?" or "Learn more with the Carbon documentation" prepositions are hard haha.

**Comment by jnm2377** [comment]
If the issue is that the current mini cards are broken, why don't we just replace with the new ones? It looks like the code for that wasn't correct and that's why it's broken.... not sure who added it, but it's not the same as the example code on the Gatsby site.

**Comment by jnm2377** [comment]
This is more of an implementation error than a component bug.

**Comment by jnm2377** [comment]
I pulled this locally and got it to work. There are 2 issues:

1. update GTC to the latest version `yarn add gatsby-theme-carbon@lastest`
2. fix the code. Here's the first example:
```jsx
<Row>
<Column colMd={5} colLg={8}>

The title should reflect the main insight the data reveals. The legend should explain the chart’s meaning by defining the association of each visual property, such as color, shape and size, to the corresponding data. When possible, use labels directly on the chart to avoid long legends. All the text should be concise and easy to understand. 
</Column>
<MiniCard
  gutterLg
  title="Learn more on Carbon"
  href="https://www.carbondesignsystem.com/data-visualization/legends"
  >
</MiniCard>
</Row>
```

**Comment by mjabbink** [comment]
@jnm2377 The mini cards are over kill in this case and the inline links makes sense as you read and then decide to “learn more” or not. I never really liked them in this use case.

**Comment by mjabbink** [comment]
@DanaPaslaru Did you create separate PR for Overview page?

**Pull request merged**